### DISCUSSION TITLE
目录与正文的“参考文献”都改成英文的“Bibliography"

### DISCUSSION THREAD

**Discussion by Sunflower0423** [nju-lug/NJUThesis#261] [Q&A]
如何目录与正文的“参考文献”都改成英文的“Bibliography"
<img width="944" alt="截屏2024-08-20 16 45 40" src="https://github.com/user-attachments/assets/fbde2cef-fa4c-401e-9af7-3c5b81f77d2b">
<img width="944" alt="截屏2024-08-20 17 06 11" src="https://github.com/user-attachments/assets/b627629f-b7b0-4278-b79e-36eb206de06e">

**Comment by atxy-blip** [comment]
```tex
\ctexset{bibname={Bibliography}}
```
加到导言区。

**Reply by Sunflower0423** [reply]
好滴谢谢！
